packages/create-instantsearch-app/src/cli/index.js

```diff
@@ -155,17 +157,17 @@ const getQuestions = ({ appName }) => ({
         new inquirer.Separator('From your index'),
         ...(await getAttributesFromIndex(answers)),
       ],
-      filter: (attributes) => attributes.filter(Boolean),
+      filter: attributes => attributes.filter(Boolean),
       when: ({ appId, apiKey, indexName }) =>
         attributesToDisplay.length === 0 && appId && apiKey && indexName,
     },
     {
       type: 'checkbox',
       name: 'attributesForFaceting',
-      message: 'Attributes to display',
+      message: 'Attributes for faceting',
```

Comment:
Prettier generated a larger diff than expected but this is the only change that matters.